com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,142 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=host
+  - --persist=.vscode-oss
```

Comment:
The data folder for the Insiders version is `.vscodium-insiders` (https://github.com/VSCodium/vscodium/blob/eea73cb5647e736a14d10ebef04c3333d6f115e0/prepare_vscode.sh#L141C39-L141C57).

Reply:
@daiyam, I'll fix in the [repo](https://github.com/flathub/com.vscodium.codium-insiders). Thanks!

Reply:
Is there any problem if @bbhtt add you as a maintainer on the flatpak repo? I asked about add @GitMensch, but he is not on VSCodium organization.